src/com/android/launcher3/util/window/WindowManagerProxy.java

```diff
@@ -127,11 +127,17 @@ public boolean isInDesktopMode() {
      * normalization
      */
     public WindowBounds getRealBounds(Context displayInfoContext, CachedDisplayInfo info) {
-        WindowMetrics windowMetrics = displayInfoContext.getSystemService(WindowManager.class)
-                .getMaximumWindowMetrics();
+        WindowMetrics windowMetrics;
+        if (Utilities.ATLEAST_R) {
+            windowMetrics = displayInfoContext.getSystemService(WindowManager.class)
+                    .getMaximumWindowMetrics();
+        
```

Comment:
Can we use the [LawnchairWindowManagerProxy](https://github.com/LawnchairLauncher/lawnchair/blob/b86b188307eccde278e0761b7bc7724ffe05ae5c/lawnchair/src/app/lawnchair/util/LawnchairWindowManagerProxy.kt#L14) instead of modifying this? Thanks!

Reply:
if you are talking about the `estimateInternalDisplayBounds(context)` method i tried here, but the return type is different and the most i could achive was `WindowBounds` and the expected type is `WindowInfo` and i not know if is posssible and how do this conversion

Reply:
Alright, let me do it

Reply:
Sure